### ISSUE TITLE
Localizable Taxonomy Routes

### ISSUE THREAD

**Issue by jasonvarga** [statamic/cms#2334]
## Bug Description

We create taxonomy routes based on its handle. 

Aside from the slugifying issue in #2273, it appears this new way doesn't yet support localizable routes.

e.g. `/tags/rad` in English and `/le-tags/le-rad` in French (sorry I don't actually speak French)

We need a way to define the taxonomies localized "slug(s)"

**Statamic version:** 3.0.2

**Comment by ebeauchamps** [comment]
LOL

**Comment by jasonvarga** [comment]
le-lol

**Comment by verybigelephants** [comment]
#1843

**Comment by aerni** [comment]
Please open this issue again. Taxonomy terms can be localized. But the taxonomy route can't. Issue https://github.com/statamic/cms/issues/2403 is probably talking about the same thing.

I think taxonomies should have the same ability to define routes as collections do.

**Comment by duncanmcclean** [comment]
I agree, especially as it's something that was possible back in v2.

**Comment by verybigelephants** [comment]
yeah, that's the reason i have shifted away from statamic actually. v3 is simply not ready for a multilingual website

**Comment by stephenmeehanuk** [comment]
Please fix this.

**Comment by christophrumpel** [comment]
I would need that too +1

**Comment by verybigelephants** [comment]
oh boy, is this still not fixed? i'm really glad that i've switched away from statamic

**Comment by Prophe1** [comment]
need this!!!!

**Comment by adriaanglibert** [comment]
I also need this ASAP. I was so happy with the localisation but this seems like a core feature.
Any workaround in the meantime?

**Comment by christophrumpel** [comment]
We didn't do it, but we received this idea for a workaround:
https://gist.github.com/jasonvarga/4342f96b77d52fcd27dad22d88253e1f

Maybe this helps.

**Comment by GertTimmerman** [comment]
We need this also at our site, we getting now a lot of 404 from wrong urls. The only thing i can do is add redirect, but that is not the real solution

**Comment by jorisvan-ons** [comment]
Is there any news on this? This seems like a core feature for any CMS that supports multisite.

**Comment by AlexanderFalkenberg** [comment]
One of the most important features for me, personally

**Comment by jensolafkoch** [comment]
Would definitely like to see that as well! :-)
